.appveyor.yml

```diff
@@ -0,0 +1,10 @@
+build: off
```

Comment:
What does this do?

Reply:
Enables the build.

No kidding.

It disables the automatic search and execution of a "build script thing".
See: https://www.appveyor.com/docs/build-phase/
Ref: https://www.appveyor.com/docs/appveyor-yml/

Reply:
🤦‍♂️